### DISCUSSION TITLE
Unable to register after 15 attempts

### DISCUSSION THREAD

**Discussion by mhz1990** [community/community#58006] [GitHub Education]
Hi everyone,
I don't know what to say, I've been trying to register for my student account all day and have literally been rejected 15 times. I've gotten the error about your profile not complete/wrong name countless times even though I have my full name and every field filled. I attend school online at Embry-Riddle Aeronautical University for my MS in information security but whenever I show any school documents( and I've tried transcripts, enrollment documents, acceptance letters), I keep getting the "nothing on that document says your classes are online" even though it says online in big bold letters. Not to mention, I now live in the Philippines (used to live in the US) which also throws the previous error ("you don't appear to be at same location as your school but the papers don't say remote"). I tried my student ID too but Embry doesn't put dates on their IDs so negative. Oh, did I mention the it won't recognize my student email (@my.erau.edu) and keeps recommending (@erau.edu) even though those emails are reserved for staff? It's pretty frustrating but the benefits of the student program are really good so I'm still trying. Thanks for your help!

**Comment by Sandro642** [comment]
Hey @mhz1990 

Contact the university's support service for assistance.
Clearly explain your situation and provide all the necessary information.
Seek help from your program advisor or academic counselor.
Verify the university's specific requirements for online registration.
Search online resources such as student forums for additional advice.
If necessary, consider visiting the university campus in person to resolve the issues.
I hope these suggestions will help you resolve your problems. Good luck!

**Reply by mhz1990** [reply]
Thanks for the feedback. I know though this really isn't an issue with my school, per se. This is a Github issue. I can try reaching out to my school though, maybe they can make a document for me just to meet Github's requirements. Thanks for the response!

**Reply by Sandro642** [reply]
if you can answered this responses if it can helpcan you answer if the answer helps you ;)

**Comment by samus-aran** [comment]
Hi @mhz1990 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 This I suspect would be most helpful ⭐ 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Lastly if you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
